fe/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -483,9 +483,8 @@
      * After dropping database(table/partition), you can recover it by using RECOVER stmt.
      * And this specifies the maximal data retention time. After time, the data will be deleted permanently.
      */
-    // TODO(ygl): temp modify it for test !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
     @ConfField(mutable = true, masterOnly = true)
-    public static long catalog_trash_expire_second = 10L; // 1day
```

Comment:
you can do test by using modify configuration file.

Reply:
In the past, it is one day。。。。